toponetx/classes/path_complex.py

```diff
@@ -0,0 +1,502 @@
+"""Path complex."""
+from collections.abc import Hashable, Iterable, Iterator
+from itertools import chain
+from typing import List, Set, Tuple, Union
```

Comment:
See previous comment, you don't need any of these classes from the `typing` module.